README.md

```diff
@@ -87,6 +87,10 @@ make tilt-up
 
 Notice that local images are build and pushed to docker running inside the minikube cluster since we are running `eval $(minikube docker-env)` before launching Tilt.
 
+### Dashboard
+
+You can import an already pre-constructed dashboard by importing this [dashboard.json](/assets/dashboard.json) file. Note that for the import to work you need to replace the json placeholder `YOUR_ACCOUNT_ID` for your New Relic `accountID` without quotes.
```

Comment:
I just followed this link and copying the dashboard JSON adds the `accountId`.  I tried as well with just removing each line regarding `accountId` in the JSON but it failed when importing with the error `Failed to create dashboard` (super useful). I'll recheck the docs again in case I missed something...

Reply:
cool, otherwise maybe a short helper command with `sed` could be useful

Reply:
Done in 24ee758e09b2f9faad79c39a84236dae8edc0da1.